### PR TITLE
Malyavin/3 y axis plot

### PR DESCRIPTION
**Pull request by otecMuxah**

### CODE REVIEW

File: web/olga/charts/static/charts/graphs.js [hunk 1]
```diff
@@ -38,23 +38,89 @@
...
-            title: chart_title
-          }
+            xaxis: {
+                title: chart_title,
+
+            },
+            yaxis: {
+                nticks: 4,
+                tickfont: {color: '#70A3FF'},
+                showline: true
```
Comment (idegtiarov) [inline web/olga/charts/static/charts/graphs.js]: Trailing comma is nice to have after the last element in the data container.

File: web/olga/charts/static/charts/graphs.js [hunk 2]
```diff
@@ -38,23 +38,89 @@
...
+            },
+            yaxis2: {
+                nticks: 4,
+                anchor: 'x',
+                overlaying: 'y',
+                side: 'right',
+                tickfont: {color: '#8BB22A'},
+                showline: true
```
Comment (idegtiarov) [inline web/olga/charts/static/charts/graphs.js]: ditto

File: web/olga/charts/static/charts/graphs.js [hunk 3]
```diff
@@ -38,23 +38,89 @@
...
+            },
+            yaxis4: {
+                nticks: 4,
+                anchor: 'x',
+                overlaying: 'y',
+                side: 'right',
+                tickfont: {color: '#CC4630'}
```
Comment (idegtiarov) [inline web/olga/charts/static/charts/graphs.js]: ditto

File: web/olga/charts/static/charts/graphs.js [hunk 4]
```diff
@@ -38,23 +38,89 @@
...
+            },
+            legend: {
+                x: 0,
+                y: 100
```
Comment (idegtiarov) [inline web/olga/charts/static/charts/graphs.js]: ditto

File: web/olga/charts/static/charts/graphs.js [hunk 5]
```diff
@@ -38,23 +38,89 @@
...
+            name: 'Instance',
+            line: {
+                color: '#70A3FF',
+                width: 2.3,
+                smoothing: 1.25
+            },
+            hovermode:'closest',
+            hoverdistance:1000,
+            spikedistance:1000,
+            type: 'scatter'
```
Comment (idegtiarov) [inline web/olga/charts/static/charts/graphs.js]: ditto

File: web/olga/charts/static/charts/graphs.js [hunk 6]
```diff
@@ -38,23 +38,89 @@
...
+            x: timeline,
+            y: chart_data[1],
+            mode: 'lines',
+            name: 'Courses',
+            line: {
+                shape: 'hv',
+                color: '#8BB22A',
+            },
+            yaxis: 'y2',
+            type: 'scatter'
```
Comment (idegtiarov) [inline web/olga/charts/static/charts/graphs.js]: ditto

File: web/olga/charts/static/charts/graphs.js [hunk 7]
```diff
@@ -38,23 +38,89 @@
...
             type: 'scatter'
-          }
-        ], layout, {displayModeBar: false});
+        };
+
+
+        let data = [
+            traceInstance,
+            traceCourses,
+            traceStudents
```
Comment (idegtiarov) [inline web/olga/charts/static/charts/graphs.js]: ditto

**Review by idegtiarov — CHANGES_REQUESTED** [review]
Some style nits are required to be fixed.

File: web/olga/charts/static/charts/graphs.js (hunk 1 above)
Comment (otecMuxah) [reply web/olga/charts/static/charts/graphs.js]: All done

File: web/olga/charts/static/charts/graphs.js (hunk 2 above)
Comment (otecMuxah) [reply web/olga/charts/static/charts/graphs.js]: done

File: web/olga/charts/static/charts/graphs.js (hunk 4 above)
Comment (otecMuxah) [reply web/olga/charts/static/charts/graphs.js]: done

File: web/olga/charts/static/charts/graphs.js (hunk 5 above)
Comment (otecMuxah) [reply web/olga/charts/static/charts/graphs.js]: done

File: web/olga/charts/static/charts/graphs.js (hunk 6 above)
Comment (otecMuxah) [reply web/olga/charts/static/charts/graphs.js]: done

File: web/olga/charts/static/charts/graphs.js (hunk 7 above)
Comment (otecMuxah) [reply web/olga/charts/static/charts/graphs.js]: done

**Review by idegtiarov — APPROVED** [review]

**Pull request merged**